sim/pokemon.ts

```diff
@@ -966,7 +968,7 @@ export class Pokemon {
 		this.apparentType = rawTemplate.types.join('/');
 		this.addedType = template.addedType || '';
 		this.knownType = true;
-		if (this.battle.gen >= 7) this.removeVolatile('autotomize');
```

Comment:
um is this ok?

Reply:
@urkerab - can you confirm this was intended?

Reply:
We had a conversation about it since then. Autotomize is no longer a volatile, it now just manually changes volatile weight.